src/mui/input/SelectManyInput.js

```diff
@@ -0,0 +1,167 @@
+import React, { Component, PropTypes } from 'react';
+import ChipInput from 'material-ui-chip-input';
+import FieldTitle from '../../util/FieldTitle';
+
+/**
+ * An Input component for a array
+ *
+ * @example
+ * <SelectManyInput source="first_name" />
+ *
+ * Pass possible options as an array of objects in the 'choices' attribute.
+ * @example
+ * const choices = [ 'Male', 'Female' ];
+ * <SelectManyInput source="gender" choices={choices} />
+ *
+ * You might want to pass the type='object', instead of 'text'. If then, the options are built from:
+ *  - the 'id' property as the option value,
+ *  - the 'name' property an the option text
+ * @example
+ * const choices = [
+ *    { id: 'M', name: 'Male' },
+ *    { id: 'F', name: 'Female' },
+ * ];
+ * <SelectManyInput source="gender" type="object" choices={choices} />
+ *
+ * The `type="embedded"` will result the array of choices with full json object inside.
+ * If you just want the result value is array of ids, please use `type="object"`.
+ * @example
+ * const choices = [
+ *    { id: 'M', name: 'Male' },
+ *    { id: 'F', name: 'Female' },
+ * ];
+ * <SelectManyInput source="gender" type="embedded" choices={choices} />
+ *
+ * Values => ['M']
+ *
+ * You can also customize the properties to use for the option name and value,
+ * thanks to the 'optionText' and 'optionValue' attributes.
+ * @example
+ * const choices = [
+ *    { _id: 123, full_name: 'Leo Tolstoi', sex: 'M' },
+ *    { _id: 456, full_name: 'Jane Austen', sex: 'F' },
+ * ];
+ * <SelectManyInput source="author_id" type="embedded" choices={choices} optionText="full_name" optionValue="_id" />
+ *
+ * The object passed as `options` props is passed to the material-ui-chip-input component
+ * @see https://github.com/TeamWertarbyte/material-ui-chip-input
+ */
+export class SelectManyInput extends Component {
+    // Comment input onBlur because SelectManyInput don't pass correct values on this event
```

Comment:
I don't understand, can you elaborate? Not catching `onBlur` or `onFocus` from the child component will probably break redux-form validation